util/generate-efs-substack.py

```diff
@@ -0,0 +1,64 @@
+from troposphere.efs import FileSystem, MountTarget
+from troposphere import Parameter, Condition, Ref, Template, Select, If, Equals, And, Not, NoValue, Output
+
+t = Template()
+EFS_options = t.add_parameter(Parameter("EFSoptions",
+                                        Type="CommaDelimitedList",
+                                        Description = "Comma separated list of efs related options, "
+                                                      "8 parameters in total, "
+                                                      "["
+                                                      "0 shared_dir,"
+                                                      "1 efs_fs_id,"
+                                                      "2 performance_mode,"
+                                                      "3 efs_kms_key_id,"
+                                                      "4 provisioned_throughput,"
+                                                      "5 encrypted,"
+                                                      "6 throughput_mode,"
+                                                      "7 existsValidMT]"))
+ComputeSecurityGroup = t.add_parameter(Parameter("ComputeSecurityGroup",
+                                                 Type="String",
+                                                 Description="SecurityGroup for Mount Target"))
+SubnetId = t.add_parameter(Parameter("SubnetId",
```

Comment:
PEP8: Variable names should be lowercase, with words separated by underscores:
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

e.g. `SubnetId --> _subnet_id`